doc/modules/ROOT/pages/backend/metadata.adoc

```diff
@@ -0,0 +1,45 @@
+= Metadata
+:sectnums:
+:sectnumlevels: 4
+:toc:
+:toclevels: 4
+:experimental:
+:keywords: AsciiDoc
+:source-highlighter: highlight.js
+:icons: font
+:imagesdir: ../../assets/images
+
+The nature https://github.com/OpenEMS/openems/blob/36343dde95c832d80ec55177b412427480973b45/io.openems.backend.common/src/io/openems/backend/common/metadata/Metadata.java#L32[Metadata]
```

Comment:
I prefer to use the term 'Nature' only for Java interfaces, that also have Channel-IDs. (see https://openems.github.io/openems.io/openems/latest/coreconcepts.html#_nature). `Metadata` ist just a normal interface.